contracts/interfaces/IOperators.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.9;
+
+interface IOperators {
```

Comment:
add high level docs to the interface and keep implementation docs for implementation specific

Reply:
added